src/common/Types.ts

```diff
@@ -967,7 +967,7 @@ export type CustomNavigatorSceneConfig = {
 };
 
 export interface NavigatorProps extends CommonProps {
-    renderScene?: (route: NavigatorRoute) => JSX.Element;
+    renderScene?: (route: NavigatorRoute, navigator: RX.Navigator<any>) => JSX.Element;
```

Comment:
At the time I began experimenting with `reactxp <Navigator />` there was no documentation available so I based my code on the `react-native <Navigator/>` component which included this second parameter. Looking at the `reactxp` documentation that has since been added I see how you've handled this differently.


In lieu of supporting the `initialRoute` property you're example code uses the following logic.
It appears this example would work for the `initialRouteStack` property as well.
```
componentDidMount() {
        // Now that the app is mounted, specify the initial
        // navigator route.
        this._navigator.immediatelyResetRouteStack([{
            routeId: NavigationRouteId.MainPanel,
            sceneConfigType: RX.Types.NavigatorSceneConfigType.Fade
        }]);
}
```